packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from "@angular/core";
+
+@Component({
+    selector: "nui-wizard-with-separate-step-headings-example",
+    templateUrl: "./wizard-with-separate-step-headings-example.component.html",
+})
+export class WizardWithSeparateStepHeadingsExampleComponent {
+    public hint = $localize `example-hint`;
```

Comment:
Maybe this. Also, there's no need to try to localize a user input value.

```suggestion
    public textboxValue = "Example Textbox Value";
```